expansion/src/main/java/no/unit/nva/expansion/model/CollaborationType.java

```diff
@@ -0,0 +1,15 @@
+package no.unit.nva.expansion.model;
+
+import lombok.Getter;
+
+@Getter
+public enum CollaborationType {
+
+    COLLABORATIVE("Collaborative"), NON_COLLABORATIVE("NonCollaborative");
```

Comment:
Tror det trengs noe default constructor for at Jackson ikke skal begynne å klage.